frontend/src/store/index.js

```diff
@@ -386,7 +386,11 @@ const getters = {
     return (cloudProfileName, region) => {
       const cloudProfile = getters.cloudProfileByName(cloudProfileName)
       const floatingPools = get(cloudProfile, 'data.providerConfig.constraints.floatingPools')
-      const availableFloatingPools = filter(floatingPools, matchesPropertyOrEmpty('region', region))
+      let availableFloatingPools = filter(floatingPools, matchesPropertyOrEmpty('region', region))
+      availableFloatingPools = filter(floatingPools, fp => {
+        const wildcard = startsWith(fp.name, '*') || endsWith(fp.name, '*')
```

Comment:
what about importing startsWith and endsWith?

Reply:
good point